src/library_syscall.js

```diff
@@ -1365,9 +1366,9 @@ var SyscallsLibrary = {
   // library_wasi.js.
 
 #if SYSCALLS_REQUIRE_FILESYSTEM == 0 && (!MINIMAL_RUNTIME || EXIT_RUNTIME)
-  $flush_NO_FILESYSTEM: function() {
+  $flush_NO_FILESYSTEM: function(stream) {
     // flush anything remaining in the buffers during shutdown
-    if (typeof _fflush !== 'undefined') _fflush(0);
+    if (typeof _fflush !== 'undefined') _fflush(stream);
```

Comment:
The 0 was intended here - this is called to flush all streams, and for nothing else.

I think it might be shorter to tell closure that the param is ignored? With this change it will grow by at least one byte (to declare the new parameter, even if it's minified to `function(a) {..`.

Reply:
Ok, I'll leave this out and revisit in a later PR.

Reply:
I think we agreed to leave this out, but it isn't left out yet?

I am ok with this change if it doesn't regress code size. My only concern is that it isn't necessary, and may increase code size.

Reply:
Ohh, now I realize I must have had a force-push accident between my work and home computers - I remember I did leave this out, but then the change (along with that other thing that you mentioned about not being done) vanished. I think I had force-pushed at work over changes that I had fixed up at home. Sorry about that!